base.cfg

```diff
@@ -113,7 +113,7 @@ programs =
     100 zope ${buildout:directory}/bin/instance [console] ${instance:location} true
 
 [alembic.ini]
-recipe = collective.recipe.template
```

Comment:
I am missing some context.
It seems to me the template  is not a genshi template https://github.com/EU-OSHA/oira.application.buildout/blob/e28678430db9c0a008a94253a2b2213ba3549fd2/templates/alembic.ini.
Why should we have this patch.

Reply:
The problem this solves is the error mentioned in the description. I got it today when rebuilding my virtualenv, and [you reported it a month ago](https://mattermost.syslab.com/syslab/pl/34a9toz1rbbxx8pkomizdyp4ue) as well. My guess is that if a buildout section that uses the recipe without the `genshi` extra runs first, then the recipe is not reinstalled later for a section that does use it.

Come to think of it, this would be a buildout bug. I should probably report it.

Reply:
I am not sure... in all the other oira buildouts I have this is not an issue.
Also on my local installation I have not this issue...
It seems to me this workaround is masking some other problem

Reply:
BTW the section I had problem with was `alembic-statistics.ini`:
```
...
While:
  Installing.
  Getting section alembic-statistics.ini.
  Initializing section alembic-statistics.ini
...
```
which is not even mentioned in this PR

Reply:
> Come to think of it, this would be a buildout bug. I should probably report it.

Yep, maybe providing a minimal cfg to show the issue.

So the problem might be solved by just reordering the parts?